### ISSUE TITLE
Re train with new quadrate training data

### ISSUE THREAD

**Issue by richard-lane** [JGIBristol/zebrafish_jaw_segmentation#55]
the labels in training sets 1/2 are currently inconsistent; training set 1 labels all 4 bones, including some of the quadrate; training set 2 labels only a subset of this. We can't just take some of the labels from Training set 1 as they dont match up exactly with training set 2, so we need to re segment these (I think this has been done), upload to to the RDSF and then re train the model.
